recipes/nasm/all/conanfile.py

```diff
@@ -49,12 +51,8 @@ def source(self):
         get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
 
     def generate(self):
-        env = VirtualBuildEnv(self)
-        env.generate()
```

Comment:
must not be removed

Reply:
Hi @SpaceIm,

What purpose does this serve when doing automated builds? They seem to work fine without it and I was just trying to eliminate extraneous code. Thanks

Reply:
Restored as it seems to cause no harm but still unclear on why it is needed when things appear to work fine without it.